doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3767,14 +3782,11 @@ automatically created.
 
       .. deprecated:: 8.10
 
-   .. cmdv:: Local Hint @hint_definition : {+ @ident}
+    .. cmdv:: Local Hint @hint_definition : {+ @ident}
```

Comment:
Or maybe it is this additional space and the one below.

Anyway, I suggest that you remove entirely this command variant and its description.